common/src/main/scala/skinny/ParamType.scala

```diff
@@ -79,22 +79,24 @@ object ParamType {
     case v: Short => scala.math.BigDecimal(v)
   })
   case object String extends AbstractParamType({
+    case v: String if v.isEmpty => null // don't trim
```

Comment:
Hmm, We often make column `not-null` as like `comment` and accept empty String.
In this case, can we convert None to empty String?